app/pages/cif/project/[project]/index.tsx

```diff
@@ -1,29 +1,145 @@
 import DefaultLayout from "components/Layout/DefaultLayout";
 import { withRelay, RelayProps } from "relay-nextjs";
 import { graphql, usePreloadedQuery } from "react-relay/hooks";
-import { ProjectOverwiewQuery } from "__generated__/ProjectOverwiewQuery.graphql";
+import { ProjectOverviewQuery } from "__generated__/ProjectOverviewQuery.graphql";
 import withRelayOptions from "lib/relay/withRelayOptions";
 
-const pageQuery = graphql`
-  query ProjectOverwiewQuery($project: ID!) {
+export const pageQuery = graphql`
+  query ProjectOverviewQuery($project: ID!) {
     session {
       ...DefaultLayout_session
     }
     project(id: $project) {
       projectName
+      rfpNumber
+      totalFundingRequest
+      summary
+      operatorByOperatorId {
+        legalName
+        bcRegistryId
+        tradeName
+      }
+      fundingStreamRfpByFundingStreamRfpId {
+        year
+        fundingStreamByFundingStreamId {
+          description
+        }
+      }
+      projectStatusByProjectStatusId {
+        name
+      }
+      contactsByProjectContactProjectIdAndContactId {
+        edges {
+          node {
+            id
+            familyName
+            givenName
+          }
+        }
+      }
+      projectManagersByProjectId {
+        edges {
+          node {
+            cifUserByCifUserId {
+              firstName
+              lastName
+              id
+            }
+          }
+        }
+      }
     }
   }
 `;
 
-function ProjectOverview({
+export function ProjectViewPage({
   preloadedQuery,
-}: RelayProps<{}, ProjectOverwiewQuery>) {
+}: RelayProps<{}, ProjectOverviewQuery>) {
   const { session, project } = usePreloadedQuery(pageQuery, preloadedQuery);
   return (
     <DefaultLayout session={session}>
-      <h2>{project.projectName}</h2>
+      <header>
+        <h2>Project Details</h2>
+      </header>
+      <dl>
+        <dt>RFP Number</dt>
+        <dd>{project.rfpNumber}</dd>
+
+        <dt>Project Name</dt>
+        <dd>{project.projectName}</dd>
+
+        <dt>Total Funding Request</dt>
+        <dd>{project.totalFundingRequest}</dd>
+
+        <dt>Summary</dt>
+        <dd>{project.summary}</dd>
+
+        <dt>Legal Operator Name and BC Registry ID</dt>
+        <dd>
+          {project.operatorByOperatorId.legalName} (
+          {project.operatorByOperatorId.bcRegistryId})
+        </dd>
+
+        <dt>Trade Name</dt>
+        <dd>{project.operatorByOperatorId.tradeName}</dd>
+
+        <dt>Funding Stream</dt>
+        <dd>
+          {
+            project.fundingStreamRfpByFundingStreamRfpId
+              .fundingStreamByFundingStreamId.description
+          }
+        </dd>
+
+        <dt>Funding Stream RFP</dt>
+        <dd>{project.fundingStreamRfpByFundingStreamRfpId.year}</dd>
```

Comment:
Have I got the correct data here? On the form it looked like the dropdown for Funding Stream RFP is just a list of years